src/main/resources/l10n/JabRef_en.properties

```diff
@@ -361,6 +361,8 @@ File\ not\ found=File not found
 
 Filter=Filter
 
+Filter\ groups=Filter groups
```

Comment:
Thanks for the fix, unfortunately this will be automatically overwritten by crowdin for non English translations. This means you have to re enter the translations using crowdin.

Reply:
Is this really how it goes? I thought crowdin is intelligent enough to pick these up (at least the feature is called "sync")

Reply:
For english this is correct, but not for other languages. Had the same problem with Italian recently. Only the english properties file is the master file. The other are translated files and therefore only synced one way from crowdin back to github.
This is also the case for new languages. you have to create new langs first in crowdin.